Library/Formula/zbar.rb

```diff
@@ -14,10 +16,15 @@ class Zbar < Formula
   depends_on 'xz'
   depends_on 'freetype'
   depends_on 'libtool' => :run
+  depends_on 'qt' => :optional
 
   # Fix JPEG handling using patch from
   # http://sourceforge.net/p/zbar/discussion/664596/thread/58b8d79b#8f67
   # already applied upstream but not present in the 0.10 release
+  #
+  # Fix building zbar with Qt4 support enabled if --with-qt is used
+  # x11Info() is only available for X11 based systems (like Linux)
+  # building with Qt4 support is not enabled by default.
```

Comment:
fair enough, I will just leave both MR open so I can redirect people wanting to use zbar on OSX with Qt4 support to them.

this is indeed needed in one of my projects, so I need to tell people how to patch their zbar.rb somehow